recreate_test.go

```diff
@@ -97,13 +101,20 @@ func TestRecreateSchema(t *testing.T) {
 				t.Fatal("recreate schema", err)
 			}
 
+			dump = trimSchema(dump)
+			if *updateGolden {
+				if err := ioutil.WriteFile(test.Golden, []byte(dump), 0644); err != nil {
+					t.Fatal(err)
+				}
+			}
+
```

Comment:
Why are these changes here???

Reply:
To make it possible to update golden files and to make test properly match new ToCQL() return